frontend/src/constants.js

```diff
@@ -41,7 +43,7 @@ const PREFIX_ROOM_SERVICE = '/api/v1/room'
 export const URL_ROOM_SERVICE = URI_ROOM_SERVICE + PREFIX_ROOM_SERVICE
 
 export const PEERJS_HOST =
-  process.env.NODE_ENV === 'production' ? '' : 'localhost'
+  process.env.NODE_ENV === 'production' ? 'localhost' : 'localhost'

```

Comment:
If left blank will get an error as the empty string is used as the host directly
![image](https://user-images.githubusercontent.com/50147457/200120631-0a428f4a-411c-4d31-92d7-f7640ec2ff3c.png)

Reply:
If you can follow the style of other services would be great, because port 80 will be assumed when omitted